### ISSUE TITLE
Big files crush page

### ISSUE THREAD

**Issue by mattermoran** [ipfs-shipyard/ipfs-webui#614]
Uploading big files > 3gb via webui crushes the page in chrome.

**Comment by olizilla** [comment]
This is still an issue. It's blocked on https://github.com/ipfs/js-ipfs-api/issues/842

I'm closing this in favour of #762
